### PR TITLE
chore: adding api-dataproc team to codeowners

### PR DESCRIPTION
**Pull request by bradmiro**
Adding @googlepapis/api-dataproc team to codeowners

### CODE REVIEW

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -6,4 +6,4 @@
 
 
 # The yoshi-nodejs team is the default owner for nodejs repositories.
-*     @googleapis/yoshi-nodejs
```
Comment (bcoe) [inline .github/CODEOWNERS:9]: I believe we've been doing this in the reverse order:

https://github.com/googleapis/nodejs-storage/blob/master/.github/CODEOWNERS#L10

`@googleapis/yoshi-nodejs @googleapis/api-dataproc`

Mind swapping for the sake of consistency (I'm not sure whether it makes a functional difference).

**Review by bcoe — CHANGES_REQUESTED** [review]

File: .github/CODEOWNERS (hunk 1 above)
Comment (meredithslota) [reply .github/CODEOWNERS:9]: Done, thank you for the suggestion.

File: .github/CODEOWNERS [hunk 2]
```diff
@@ -6,4 +6,4 @@
...
+*     @googleapis/yoshi-nodejs @googleapis/api-dataproc
```
Comment (JustinBeckwith) [inline .github/CODEOWNERS:9]: Forgive me - this is controlled now in `.repo-metadata.json` in the root of the repo, and it uses that to generate the `CODEOWNERS` file now.  Kinda like this:
https://github.com/googleapis/nodejs-datastore/blob/master/.repo-metadata.json#L12

Comment (meredithslota) [reply .github/CODEOWNERS:9]: Does it? I don't see anything similar here: https://github.com/googleapis/nodejs-dataproc/blob/master/.repo-metadata.json even for the existing CODEOWNER. How should we structure this if there are multiple teams here, e.g. Yoshi as primary and Dataproc team as secondary?

Comment (bcoe) [reply .github/CODEOWNERS:9]: The @googleapis/yoshi-nodejs team should always be added, with an optional second team:

**Review by bcoe — APPROVED** [review]

**Pull request merged**